<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output table</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>37</y>
     <width>701</width>
     <height>364</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Discharge</string>
    </attribute>
    <widget class="QTableWidget" name="tbldischarge">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>691</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Depth</string>
    </attribute>
    <widget class="QTableWidget" name="tbldepth">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>691</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Observed data</string>
    </attribute>
    <widget class="QTableWidget" name="tblobserveddata">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>691</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Arial&quot;;</string>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>413</y>
     <width>701</width>
     <height>160</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Discharge summary</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>3</y>
       <width>691</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Depth summary</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open MDB</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMdb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnViwfileQ">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View text file(Q)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnViwfileT">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View text file(T)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
